tools/importer/import.js

```diff
@@ -185,6 +185,26 @@ function setPlayIcons(main, document) {
   });
 }
 
+// javascript function to fix heading levels in a page, to not skip levels
+function fixHeadings(rootEl) {
+  const headings = [...rootEl.querySelectorAll('h2, h3, h4, h5, h6')];
+  headings.forEach((heading) => {
+    console.log(heading.outerHTML);
+  });
+  const maxLevel = headings.reduceRight((max, heading) => {
+    const level = parseInt(heading.tagName.charAt(1), 10);
+    return (level > 1 &&  level < max) ? level : max;
+  }, 6);
+  console.log(`maxLevel: ${maxLevel}`);
+  const levelGap = maxLevel - 2;
```

Comment:
in this case max heading will be h2 in the document, we still need h1 right?
possibly we need a map like below?

const levelMap = {
    'h1': 'h1', 
    'h3': 'h2',
    'h4': 'h3'   
  };

Reply:
The idea of the function is that we always except H1 to be present so we skip it. Then, the function check what is the biggest heading in the page and from this, calculate how many we must "increase" each heading to keep the order sequence.

Reply:
thanks @catalan-adobe , is there document link which you tested with this change ?
rest looks good.